shishito/library/modules/runtime/environment/browserstack_control_environment.py

```diff
@@ -11,6 +9,19 @@
 class ControlEnvironment(ShishitoEnvironment):
     """ Browserstack control environment. """
 
+    CAPABILITIES = (
```

Comment:
i think it's better use dict or namedtuple 

but at all I am not sure if new declarative way is better then old one
In old one i am directlu see what's going. In new one it's harder and code is in fact little bit longer

if you use old code with eg shortcut to `self.shishito_support.gid` it would be imho better
```gid = self.shishito_support.gid
capabilities = {
   'build': gid('build_name')
   ...
}